src/category_theory/limits/shapes/equalizers.lean

```diff
@@ -172,14 +239,82 @@ abbreviation equalizer := limit (parallel_pair f g)
 abbreviation equalizer.ι : equalizer f g ⟶ X :=
 limit.π (parallel_pair f g) zero
 
-@[simp, reassoc] lemma equalizer.condition : equalizer.ι f g ≫ f = equalizer.ι f g ≫ g :=
+lemma equalizer.ι.fork : fork.ι (limits.limit.cone (parallel_pair f g)) = equalizer.ι f g := rfl
+
+lemma equalizer.ι.eq_app_zero : (limit.cone (parallel_pair f g)).π.app zero = equalizer.ι f g := rfl
+
+@[reassoc] lemma equalizer.condition : equalizer.ι f g ≫ f = equalizer.ι f g ≫ g :=
 begin
   erw limit.w (parallel_pair f g) walking_parallel_pair_hom.left,
   erw limit.w (parallel_pair f g) walking_parallel_pair_hom.right
 end
 
 abbreviation equalizer.lift {W : C} (k : W ⟶ X) (h : k ≫ f = k ≫ g) : W ⟶ equalizer f g :=
 limit.lift (parallel_pair f g) (fork.of_ι k h)
+
+/-- A specialization of the uniqueness statement about limits to equalizers -/
+lemma equalizer.lift.uniq {W : C} (k : W ⟶ X) (h : k ≫ f = k ≫ g) (l : W ⟶ equalizer f g)
```

Comment:
Two points:
(1) The docstring should make sense as a stand-alone statement, out of the context of this file. This one isn't too bad, but still, if a user reads this in some pop-up message, it also doesn't give them that much info. Long docstrings are *not* discouraged (-;
(2) Once again, wouldn't it make more sense to reverse the direction of this statement? I think it would make a good simp-lemma in that direction.

Reply:
Regarding your second point: I cannot really see how simp should work here. The intended use case of this lemma is to show that two maps are equal by noticing that they are both equalizer lifts of the same cone (see for example the proof of `equalizer.ι_mono` below). Thus, the way the conclusion is written down is really the way that reflects the mathematical meaning of the lemma (and, arguably, in a mathematician's mind, the `l` might be the "complicated" expression compared to `equalizer.lift f g k h`).

I will try to rewrite the docstring (possibly including some of what I have just said).

Reply:
Ok, fair enough. Thanks for explaining.